src/main/java/com/thoughtworks/pacman/core/Game.java

```diff
@@ -14,6 +14,10 @@
     private final Ghosts ghosts;
     private final PacmanTileVisitor pacmanTileVisitor;
 
+    private boolean isItOnPause;
+
+    public int Score;
```

Comment:
Why do you use the capital S for the variable name? It does not fit with the style of the existing code.

Rename `Score` => `score`